### ISSUE TITLE
Tk status in TexText 1.0 

### ISSUE THREAD

**Issue by sizmailov** [textext/textext#173]
Tk is guaranteed to be a part of every python interpreter, so TexText may run even if GTK libraries are missing. At the same time the functionality and interface are very limited compared to GTK. Now (inkscape>=1.0) guarantees that GTK3 is installed, so I think it might be a good point to deprecate Tk or remove entirely. 

Is there any user which still **need** Tk support in inkcape1.0+? (please leave a comment if you need tk and tell us what is your use case)

**Comment by sizmailov** [comment]
@jcwinkler You were against TK deprecation in TexText 1.0 as GTK was shaky in inkscape 1.0. Is current situation still same? 

I assume GTK+PyGObject are reliably available on all platforms as inkscape 1.0 release appoaches.
Inkscape's release date was shifted number of times, but now it seems to be really about release which is around 4th of May.

As a compromise we can keep Tk version in 1.x in unsupported mode and plan removal to 2.x.
In 1.0 we can place deprecation warning in Tk UI to get feedback about it, smth like:

```
TexText does not longer support Tk UI.  
Please see deprecation page notice on textext.github.com/textext/tk-deprecation
```

On deprecation page we can provide a link to this issue to and ask people who need Tk describe their use case.

**Comment by jcwinkler** [comment]
I agree that we should drop Tk support as soon as possible. However, to be on the safe side let's keep it in the 1.x branch using the approach you sketched and without developing it further.

I will add this to the doc I am also currently updating.

On Windows it is guaranteed that GTK+pyGObject is included. Currently I am building an installer providing also GTKSourceview.

On Linux the situation is quite unclear to me:

- Will gi retroinspection etc. be included as a dependency in the deb/rpm packages? Or do the user still need to install it?
- Will gi retroinspection etc. be included in the Appimages, flatpack and snap packages?
